contrib/devtools/symbol-check.py

```diff
@@ -218,6 +239,26 @@ def check_MACHO_libraries(filename) -> bool:
             ok = False
     return ok
 
+def pe_read_libraries(filename) -> List[str]:
+    p = subprocess.Popen([OBJDUMP_CMD, '-x', filename], stdout=subprocess.PIPE, stderr=subprocess.PIPE, stdin=subprocess.PIPE, universal_newlines=True)
+    (stdout, stderr) = p.communicate()
+    if p.returncode:
+        raise IOError('Error opening file')
+    libraries = []
+    for line in stdout.splitlines():
+        if 'DLL Name:' in line:
+            tokens = line.split()
+            libraries.append(tokens[2])
```

Comment:
nit: wouldn't rely on the whitespace in the left-hand side description string here, but rather use ': ' as seperator and then access with index 1.
```suggestion
            tokens = line.split(': ')
            libraries.append(tokens[1])
```

Reply:
Sure.